test/Elastic.Apm.Tests.MockApmServer/Startup.cs

```diff
@@ -37,10 +38,17 @@ public void ConfigureServices(IServiceCollection services)
 		}
 
 		// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+#if NET5_0
+		public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+#else
 		public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+#endif
 		{
 			app.UseDeveloperExceptionPage();
+
+#if !NET5_0
 			app.UseMvc();
+#endif
```

Comment:
```suggestion
#else
			app.UseRouting();
			app.UseEndpoints(endpoints => endpoints.MapControllers());
#endif
```

Looks like this is needed for net5.0